2_adventures/001/liamseanbrady/14/README.md

```diff
@@ -0,0 +1,4 @@
+### Keeping It Going
+
+- Day 13 hasn't been merged yet, so doing another pull request for day 14 which doesn't have a `13` directory yet seems weird, but apparently that is okay! I'm looking forward to reading more about this particular issue further in Day 15.
```

Comment:
it is okay! and you should be good now... [here's the wiki page](https://github.com/RubySteps/21-day-challenge/wiki/Suggested-Git-Workflow#you-havent-merged-my-previous-days-work-what-do-i-do) (which sounds like you've already seen)